### PR TITLE
New parallel blanket cutters

### PR DESCRIPTION
**Pull request by Shimwell**
## Proposed changes

The aim of this PR is to add another blanket cutter shape that makes it easier to create blanket bananna segments with parallel segments. Currently there is a PR in to create bananna segments with non parallel sides. This PR would reduce the amount of code needed by the end user to make[ this blanket example](https://github.com/ukaea/paramak/blob/main/examples/example_parametric_components/make_demo_style_blankets.py) 

Screenshot from 2020-10-26 22-22-10
Screenshot from 2020-10-26 22-16-31

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Pep8 applied
- [x] Unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: paramak/parametric_components/blanket_cutter_parallels.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    parrallel sides.Typically used to divide a blanket into vertical
+    sections with a fixed distance between each section.
+
+    Args:
+        distance (float): extruded distance (cm) of the cutter which translates
+            to being the gap size between blankets when the cutter is used to
+            segment blankets.
+        gap_size (float): the distance between the inner edges of the two
+            parrallel extrusions
+        azimuth_placement_angle (float, optional): the azimuth angle(s) used
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_components/blanket_cutter_parallels.py:21]: shouldn't it be `float or list` ?

Comment (Shimwell) [reply paramak/parametric_components/blanket_cutter_parallels.py:21]: ah yes thanks for that, I shall correct

Comment (Shimwell) [reply paramak/parametric_components/blanket_cutter_parallels.py]: I've added a diagram to the PR, I think gap_size is just a float. 

I had copied azimuth_placement_angle from shape.py so I have now added iterable to that docstring in this commit 
https://github.com/ukaea/paramak/pull/399/commits/f75a1cb89a93f68f606bc6b230affb21c8b5a314

Comment (RemDelaporteMathurin) [reply paramak/parametric_components/blanket_cutter_parallels.py]: You are right, I was talking about azimuth placement angle :-)

File: paramak/parametric_components/blanket_cutter_parallels.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        **kwargs
+    ):
+
+        super().__init__(
+            distance=distance,
+            stp_filename=stp_filename,
+            stl_filename=stl_filename,
+            material_tag=material_tag,
+            **kwargs
+        )
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_components/blanket_cutter_parallels.py:55]: I think `name` needs to be passed to super

File: paramak/parametric_components/blanket_cutter_parallels.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        width=2000,
+        stp_filename="BlanketCutterParallels.stp",
+        stl_filename="BlanketCutterParallels.stl",
+        name="blanket_cutter_parallels",
+        material_tag="blanket_cutter_parallels_mat",
+        **kwargs
+    ):
+
+        super().__init__(
+            distance=distance,
```
Comment (Shimwell) [inline paramak/parametric_components/blanket_cutter_parallels.py:50]:
```suggestion
            name=name,
            distance=distance,
```

File: paramak/parametric_components/blanket_cutter_parallels.py (hunk 2 above)
Comment (Shimwell) [reply paramak/parametric_components/blanket_cutter_parallels.py]: added , thanks

**Comment by Shimwell** [comment]
Screenshot from 2020-10-27 16-01-55

**Comment by Shimwell** [comment]
Screenshot from 2020-10-27 16-11-36

**Pull request merged**